tests/zkevm/test_worst_bytecode.py

```diff
@@ -144,16 +156,27 @@ def generate_deploy_tx(contracts_to_deploy: int):
     post = {}
     deployed_contract_addresses = []
     for i in range(total_contracts_to_deploy):
-        deployed_contract_address = compute_create_address(
+        deployed_contract_address = compute_create2_address(
             address=factory_address,
-            nonce=i + 1,
+            salt=i,
+            initcode=initcode,
         )
         post[deployed_contract_address] = Account(nonce=1)
         deployed_contract_addresses.append(deployed_contract_address)
 
     opcode_code = (
-        sum(Op.POP(opcode(address=address)) for address in deployed_contract_addresses) + Op.STOP
+        # Setup memory for later CREATE2 address generation loop.
+        Op.MSTORE(0, factory_address)
+        + Op.MSTORE8(32 - 20 - 1, 0xFF)  # 0xFF prefix byte
+        + Op.MSTORE(32, 0)
+        + Op.MSTORE(64, initcode.keccak256())
+        # Main loop
+        + While(
+            body=Op.EXTCODESIZE(Op.SHA3(32 - 20 - 1, 85)) + Op.MSTORE(32, Op.ADD(Op.MLOAD(32), 1)),
+            condition=Op.PUSH1(1),
```

Comment:
Run this loop forever using all available gas.
This `condition=` isn't super ideal since `While` does some `JUMPI`.

I could avoid using `While` and do `JUMPDEST+loop+JUMP` but it is a bit messy since I would have to also do `Op.PC` and do some calcs for the right jump target. I think usin `While` is simply much better and understandable.